### PR TITLE
change tracker log to common logging

### PR DESCRIPTION
**Pull request by michaeltiket**
## :camera: Screenshots
image
image

## :page_facing_up: Context
- 

## :pencil: Changes
- change tracker log to common logging
- add copy each item to clipboard
- add filtering tag

## :paperclip: Related PR
- 

## :no_entry_sign: Breaking
- 

## :hammer_and_wrench: How to test
-

## :stopwatch: Next steps
-

### CODE REVIEW

File: library-extra-logging/src/main/java/com/chucker/logging/internal/data/repository/LoggingRepository.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.chucker.logging.internal.data.repository
+
+import com.chucker.logging.internal.data.entity.LogData
+
+internal interface LoggingRepository {
+
+    suspend fun insertLog(logData: LogData)
+
+    suspend fun deleteAllTransactions()
```
Comment (esafirm-tiket) [inline library-extra-logging/src/main/java/com/chucker/logging/internal/data/repository/LoggingRepository.kt]: `deleteAllLog()` ?

File: library-extra-logging/src/main/java/com/chucker/logging/internal/ui/LoggingAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        private val copyCallback: (LogData) -> Unit
+    ) : RecyclerView.ViewHolder(binding.root) {
+        fun bind(value: LogData) {
+            binding.tag.text = value.tag
+            binding.message.text = try {
+                JSONObject(value.logString).toString(2)
+            } catch (ignored: Exception) {
+                value.logString
+            }
+            binding.timeStamp.text = SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS").format(value.timeStamp)
```
Comment (esafirm-tiket) [inline library-extra-logging/src/main/java/com/chucker/logging/internal/ui/LoggingAdapter.kt]: Memoize `SimpleDateFormat`

File: library-extra-logging/src/main/java/com/chucker/logging/internal/ui/LoggingAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+internal class LoggingAdapter(private val copyCallback: (LogData) -> Unit) :
+    ListAdapter<LogData, LoggingAdapter.ViewHolder>(DiffCallback) {
+
+    internal class ViewHolder(
+        private val binding: ChuckerItemLogBinding,
+        private val copyCallback: (LogData) -> Unit
+    ) : RecyclerView.ViewHolder(binding.root) {
+        fun bind(value: LogData) {
+            binding.tag.text = value.tag
+            binding.message.text = try {
```
Comment (esafirm-tiket) [inline library-extra-logging/src/main/java/com/chucker/logging/internal/ui/LoggingAdapter.kt]: Memoize the result of this

File: settings.gradle [hunk 4]
```diff
@@ -11,6 +11,8 @@ includeModules([
         ':library-feature-flag',
         ':library-extra-activity-info',
         ':library-extra-activity-info-noop',
-        ':library-extra-tracker-log',
-        ':library-extra-tracker-log-noop'
-])
\ No newline at end of file
+//        ':library-extra-tracker-log',
```
Comment (esafirm-tiket) [inline settings.gradle]: Why?

**Review by esafirm-tiket — CHANGES_REQUESTED** [review]

File: settings.gradle (hunk 4 above)
Comment (michaeltiket) [reply settings.gradle]: ah for rename module

Comment (esafirm-tiket) [reply settings.gradle]: 👍

**Pull request merged**